index.md

```diff
@@ -128,221 +128,175 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
-| 3 | ref/component-versions | [Component versions](https://discourse.ubuntu.com/t/component-versions/21413)|
-| 2 | ref/requirements | [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) |
-| 2 | ref/appliance-command-reference/landing | [Appliance command reference](https://discourse.ubuntu.com/t/39525)|
-| 2 | ref/amc-command-reference/landing | [AMC command reference](https://discourse.ubuntu.com/t/40797) |
-| 2 | ref/provided-images | [Provided images](https://discourse.ubuntu.com/t/provided-images/24185)|
-| 2 | ref/supported-rendering-resources | [Supported rendering resources](https://discourse.ubuntu.com/t/supported-rendering-resources/37322) |
-| 2 | ref/supported-codecs | [Supported codecs](https://discourse.ubuntu.com/t/37323)|
-| 2 | ref/android-features | [Supported Android features](https://discourse.ubuntu.com/t/supported-android-features/28825)|
-| 2 | ref/ams-configuration | [AMS configuration](https://discourse.ubuntu.com/t/ams-configuration/20872)|
-| 2 | ref/application-manifest | [Application manifest](https://discourse.ubuntu.com/t/application-manifest/24197)|
-| 2 | ref/api-reference | [APIs](https://discourse.ubuntu.com/t/api-reference/24339) |
-| 3 | ref/ams-http-api | [AMS HTTP API](https://anbox-cloud.github.io/latest/ams/)|
-| 3 | ref/anbox-https-api | [Anbox HTTP API](https://discourse.ubuntu.com/t/anbox-http-api-reference/17819)|
-| 3 | ref/anbox-stream-gateway | [Stream Gateway API](https://anbox-cloud.github.io/latest/anbox-stream-gateway/)|
-| 3 | ref/anbox-platform-sdk-api | [Anbox Platform SDK API](https://anbox-cloud.github.io/latest/anbox-platform-sdk/)|
-| 2 | ref/sdks | [Anbox Cloud SDKs](https://discourse.ubuntu.com/t/anbox-cloud-sdks/17844)|
-| 2 | ref/network-ports | [Network ports](https://discourse.ubuntu.com/t/network-ports/33650)|
-| 2 | ref/addon-manifest | [Addon manifest](https://discourse.ubuntu.com/t/addons/25293)|
-| 2 | ref/hooks | [Hooks](https://discourse.ubuntu.com/t/hooks/28555)|
-| 2 | ref/webrtc-streamer | [WebRTC streamer](https://discourse.ubuntu.com/t/webrtc-streamer/30195)|
-| 2 | ref/prometheus | [Prometheus metrics](https://discourse.ubuntu.com/t/prometheus-metrics/19521)|
-| 2 | ref/perf-benchmarks | [Performance benchmarks](https://discourse.ubuntu.com/t/performance-benchmarks/24709)|
-| 2 | ref/license-information | [License information](https://discourse.ubuntu.com/t/license-information/36649) |
-| 2 | ref/glossary | [Glossary](https://discourse.ubuntu.com/t/glossary/26204)|
+| 1 | reference/landing | [Reference](https://discourse.ubuntu.com/t/reference/28828) |
+| 2 | reference/releases-versions | [Releases and versions](https://discourse.ubuntu.com/t/releases-and-versions/37993) |
+| 3 | reference/roadmap | [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359)|
+| 3 | reference/release-notes/release-notes | [Release notes](https://discourse.ubuntu.com/t/release-notes/17842)|
```

Comment:
`reference/release-notes/release-notes`

Is that intended? Shouldn't it be something like `landing`, as for the other sub directories?

Reply:
This is intended because technically the first "release-notes" is the folder name and the folder is mainly for convenience of grouping all the release notes together. "release-notes.md" is the file that actually is the release notes.